bindep.txt

```diff
@@ -17,20 +17,20 @@ python3-Cython [compile platform:fedora-35 platform:rhel-9]
 # python39-Cython [compile platform:centos-8 platform:rhel-8]
```

Comment:
Confirmed that `python3-Cython` is not available on `centos-9` platform either.

Seems it's been moved the `CRB` (CodeReady) repository, which isn't available in EEs (typically).
See: https://pkgs.org/download/python3-Cython